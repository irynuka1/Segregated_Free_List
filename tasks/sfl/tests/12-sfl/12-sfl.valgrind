==3445== Memcheck, a memory error detector
==3445== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3445== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==3445== Command: /home/irynuka1/Desktop/UNI/SDA/Tema1/sfl
==3445== Parent PID: 3444
==3445== 
==3445== 
==3445== HEAP SUMMARY:
==3445==     in use at exit: 0 bytes in 0 blocks
==3445==   total heap usage: 89 allocs, 89 frees, 9,816 bytes allocated
==3445== 
==3445== All heap blocks were freed -- no leaks are possible
==3445== 
==3445== For lists of detected and suppressed errors, rerun with: -s
==3445== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
