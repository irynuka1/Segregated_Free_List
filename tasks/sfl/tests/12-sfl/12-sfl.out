+++++DUMP+++++
Total memory: 384 bytes
Total allocated memory: 16 bytes
Total free memory: 368 bytes
Free blocks: 26
Number of allocated blocks: 2
Number of malloc calls: 2
Number of fragmentations: 0
Number of free calls: 0
Blocks with 8 bytes - 14 free block(s) : 0xcfa 0xd02 0xd0a 0xd12 0xd1a 0xd22 0xd2a 0xd32 0xd3a 0xd42 0xd4a 0xd52 0xd5a 0xd62
Blocks with 16 bytes - 8 free block(s) : 0xd6a 0xd7a 0xd8a 0xd9a 0xdaa 0xdba 0xdca 0xdda
Blocks with 32 bytes - 4 free block(s) : 0xdea 0xe0a 0xe2a 0xe4a
Allocated blocks : (0xcea - 8) (0xcf2 - 8)
-----DUMP-----
